### ISSUE TITLE
Can't receive personal message from yahoo contact.

### ISSUE THREAD

**Issue by GoogleCodeExporter** [msndevs/msnp-sharp#210]
```
What steps will reproduce the problem?
Can't receive personal message from yahoo contact.
why?

What is the expected output? What do you see instead?


What version of the product are you using? (MSNPSharp, OS, Mono etc.)
3.1.2 Beta
```

Original issue reported on code.google.com by `jarvis.chen66@gmail.com` on 6 Aug 2010 at 7:51

Attachments:
- [img.png](https://storage.googleapis.com/google-code-attachments/msnp-sharp/issue-210/comment-0/img.png)

**Comment by GoogleCodeExporter** [comment]
```
Thanks,

Fixed it in trunk version and 3.0 stable. Please don't use 3.1.2 Beta. Check 
out the code in MSNPSHARP_30_STABLE instead.
```

Original comment by `freezing...@gmail.com` on 6 Aug 2010 at 8:54
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
Original comment by `hepha...@gmail.com` on 20 Sep 2010 at 10:34
- Added labels: **Milestone-Release3.0.3**
